src/components/footer/FoodTags.tsx

```diff
@@ -0,0 +1,18 @@
+import { Chip, Typography, Box } from '@mui/material';
+import { footerTagsContainer, footerTagsTitle } from '@styles/footer-styles';
+import { food } from './constants';
+
+export default function FoodTags() {
+  return (
+    <>
+      <Typography sx={footerTagsTitle} variant="h6">
+        Product tags
+      </Typography>
+      <Box sx={footerTagsContainer}>
+        {food.map((name, i) => (
+          <Chip key={i} label={name} />
```

Comment:
use index for the key is not good. You can use useId() hook from react 18, or .name if the name is unique

Reply:
about the useId() hook I found this (it says that it's not a good practice to use it for keys):
https://twitter.com/reactjs/status/1510334505252491267?s=20&t=OXCRI-sGD6YwBJFcVrd-Ew